src/sys/signal.rs

```diff
@@ -210,6 +211,19 @@ libc_bitflags!{
     }
 }
 
+#[cfg(target_os = "android")]
```

Comment:
It's worth noting that this change is necessary because of https://github.com/rust-lang/libc/pull/511